### ISSUE TITLE
PlotAggregate - Failed plots (Warning: At least one axes collapsed to zero width or height )

### ISSUE THREAD

**Issue by YevhenAkimov** [loosolab/TOBIAS#27]
Hello, I get a plot with near-zero lengths of X and Y axes when running PlotAggregate on BINDetect results obtained using `HOCOMOCOv11_core_HUMAN_mono_jaspar_format.jaspar` but for JASPAR2020 it worked fine.
Here is the picture:
![CTCF_HUMAN H11MO 0 A_CTCF_HUMAN H11MO 0 A](https://user-images.githubusercontent.com/52507019/91465529-902f9f80-e896-11ea-807e-6c6cf18a33c8.png)

Code:

`
TOBIAS PlotAggregate --TFBS BINDetect_single_output_HO/CTCF_HUMAN.H11MO.0.A_CTCF_HUMAN.H11MO.0.A/beds/CTCF_HUMAN.H11MO.0.A_CTCF_HUMAN.H11MO.0.A_all.bed BINDetect_single_output_HO/CTCF_HUMAN.H11MO.0.A_CTCF_HUMAN.H11MO.0.A/beds/CTCF_HUMAN.H11MO.0.A_CTCF_HUMAN.H11MO.0.A_test_tobias_bindetect_HO_bound.bed BINDetect_single_output_HO/CTCF_HUMAN.H11MO.0.A_CTCF_HUMAN.H11MO.0.A/beds/CTCF_HUMAN.H11MO.0.A_CTCF_HUMAN.H11MO.0.A_test_tobias_bindetect_HO_unbound.bed --signals ATACorrect_test_genrich/final_uncorrected.bw ATACorrect_test_genrich/final_expected.bw ATACorrect_test_genrich/final_corrected.bw --output CTCF_HUMAN.H11MO.0.A_CTCF_HUMAN.H11MO.0.A.png --share_y sites --plot_boundaries
`
Truncated output:

```
...
2020-08-27 17:15:33 (9991) [INFO]	Plotting regions CTCF_HUMAN.H11MO.0.A_CTCF_HUMAN.H11MO.0.A_all from signal final_corrected
2020-08-27 17:15:33 (9991) [INFO]	Plotting regions CTCF_HUMAN.H11MO.0.A_CTCF_HUMAN.H11MO.0.A_test_tobias_bindetect_HO_bound from signal final_corrected
2020-08-27 17:15:33 (9991) [INFO]	Plotting regions CTCF_HUMAN.H11MO.0.A_CTCF_HUMAN.H11MO.0.A_test_tobias_bindetect_HO_unbound from signal final_corrected
2020-08-27 17:15:33 (9991) [INFO]	Adjusting final details
/Users/akimov/Library/Python/3.7/lib/python/site-packages/tobias/tools/plot_aggregate.py:504: UserWarning: constrained_layout not applied.  At least one axes collapsed to zero width or height.
  plt.savefig(args.output, bbox_inches='tight')

2020-08-27 17:15:37 (9991) [INFO]	Finished PlotAggregate run (total time elapsed: 0:00:20.779433)
```

Thanks!

**Comment by msbentsen** [comment]
Hi, ugh that indeed looks strange! So you are saying that you got the correct plot using .bed-files estimated from JASPAR motifs?

I can imagine that this has to do with the long names arising from HOCOMOCO, which somehow breaks the grid. I will have a look!

**Comment by YevhenAkimov** [comment]
Hi again, Yes it was due to the long names. When I shortened the names it went fine.

**Comment by msbentsen** [comment]
I have just released TOBIAS 0.11.7, which fixes this with dynamic fontsize parameter for long names (to make the names fit to the width of the plots). I am closing this issue, but let me know in case of any other issues!
